lib/elastic_apm/normalizers/active_record.rb

```diff
@@ -9,10 +9,13 @@ module ActiveRecord
       class SqlNormalizer < Normalizer
         register 'sql.active_record'
 
+        TYPE = 'db'
+        ACTION = 'sql'
+
         def initialize(*args)
           super
 
-          @type = format('db.%s.sql', lookup_adapter || 'unknown').freeze
+          @subtype = lookup_adapter || 'unknown'
```

Comment:
you're no longer calling `format` on the result of this line or freezing the string. Is that intentional/ok?

Reply:
Honestly not sure why we were freezing it before. The string has to be built every time anyway.